tests/ci/avalon-multiple-wp-direct-model-sgx-tests.yaml

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-wp-direct-model-sgx-tests:
+    image: avalon-shell:${ISOLATION_ID}
+    working_dir: "/project/avalon/"
+    volumes:
+      - ../logs:/project/avalon/logs
+    entrypoint: "bash -c \"\
+        ./tools/run_tests.sh -l avalon-listener \""
+    depends_on:
+      - avalon-enclave-manager
+      - avalon-listener
+
+  avalon-kme:
+    image: avalon-sgx-enclave-manager-kme:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 1948
+      - 5555
+    volumes:
+      # Map aesm service on host to docker, required only in HW mode.
+      - /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket
+    devices:
+      - "/dev/isgx:/dev/isgx"
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+
+  avalon-enclave-manager:
+    image: avalon-sgx-enclave-manager-wpe:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 5555
+    volumes:
+      # Map aesm service on host to docker, required only in HW mode.
+      - /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket
+    devices:
+      - "/dev/isgx:/dev/isgx"
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --kme_listener_url http://avalon-kme:1948
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+      - avalon-kme
+
+  avalon-lmdb:
+    image: avalon-lmdb:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 9090
+    command: |
+      bash -c "
+        kv_storage --bind http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+
+  avalon-listener:
+    image: avalon-listener:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 1947
+      - 5555
+    command: |
+      bash -c "
+        avalon_listener --bind http://avalon-listener:1947 --lmdb_url http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+
```

Comment:
This file has single instance of wpe and kme?

Reply:
It has 2 KME, 1 having just 1 WPE and 1 having 2 WPE.

Reply:
Addressed.